_api-reference/nodes-apis/nodes-stats.md

```diff
@@ -977,6 +979,7 @@ active | Integer | The number of active threads in the pool.
 rejected | Integer | The number of tasks that have been rejected.
 largest | Integer | The peak number of threads in the pool.
 completed | Integer | The number of tasks completed.
+total_wait_time | Integer | The total amount of time tasks spent waiting in the thread pool queue. Currently, only `search`, `search_throttled`, and `index_searcher` thread pools support this metric.
```

Comment:
@kolchfa-aws @jed326 We are saying that this is supported for `search` threadpool but in the example above it is showing up for `ad-batch-task-threadpool` and `OPENSEARCH_ML_TASK_THREAD_POOL`. So either we add search threadpool stats in above example with this new metric or remove the new metric from those threadpool